events/consts.go

```diff
@@ -35,4 +35,10 @@ const (
 
 	E_PUBLISHER_CREATE = "PUBLISHER_CREATE"
 	E_PUBLISHER_UPDATE = "PUBLISHER_UPDATE"
+
```

Comment:
Needs `DELETE` events for blog_posts and tweets

Reply:
Done